mender-convert.sh

```diff
@@ -5,28 +5,36 @@ cat << EOF
 
 Mender conversion tool
 
-A tool for taking an existing embedded image (Debian, Ubuntu, Raspbian, etc) and
-convert it to a Mender image by restructuring partition table and adding
+A tool for taking an existing embedded image (Debian, Ubuntu, Raspbian, etc)
+and convert it to a Mender image by restructuring partition table and adding
 necessary files.
 
-Usage: $0 [prepare_image | make_sdimg | install_mender | install_bootloader | make_all] [options]
+Usage: $0 [raw-disk-image-shrink-rootfs |
+           raw-disk-image-create-partitions |
+           install-mender-to-mender-disk-image |
+           install-bootloader-to-mender-disk-image |
+           from-raw-disk-image] [options]
 
 Actions:
 
-        prepare_image       - shrinks existing embedded image
-
-        make_sdimg          - composes image file compliant with Mender partition
-                              layout
-
-        install_mender      - installs Mender client related files
-
-        install_bootloader  - installs Grub related files
-
-        make_artifact       - creates Mender artifact file based on .sdimg file
-
-        make_all            - composes fully functional .sdimg file compliant
-                              with Mender partition layout, having all necessary
-                              files installed
+        from-raw-disk-image                     - composes fully functional Mender
+                                                  image compliant with Mender
+                                                  partition layout, having all
+                                                  necessary files installed
+        mender-disk-image-to-artifact           - creates Mender artifact file
+                                                  from Mender image
+
+Expert actions:
```

Comment:
Same here "Expert ACTIONS:"

Reply:
...and here "Expert commands:" I see that uppercase is commonly used in the "Usage" line, afterwards lowercase can be seen. Is it fine?

Reply:
Oh, and there is of course follow-up commit for options names.

Reply:
:+1: